rules-reviewed/quarkus/springboot/springboot-metrics-to-quarkus.windup.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-metrics-to-quarkus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot Metrics to Quarkus Microprofile Metrics extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="springboot"/>
+        <targetTechnology id="quarkus"/>
+    </metadata>
+    <rules>
+        <rule id="springboot-metrics-to-quarkus-0100">
+            <when>
+                <dependency groupId="io.micrometer" artifactId="micrometer-core"/>
+            </when>
+            <perform>
+                <hint title="Replace the Micrometer dependency with Quarkus 'metrics' extension" effort="5" category-id="mandatory">
+                    <message>
+                        Replace the Micrometer dependency with Quarkus 'metrics' extension
```

Comment:
Add 2 spaces otherwise the hint looks bad
![Screenshot from 2020-04-27 12-03-58](https://user-images.githubusercontent.com/7288588/80360257-5c5c5300-887f-11ea-9522-7ff7b3bc47ca.png)

```suggestion
                        Replace the Micrometer dependency with Quarkus 'metrics' extension.  
```

Reply:
+1